docs/clusterfuzzlite.md

```diff
@@ -14,10 +14,12 @@ ClusterFuzzLite is based on [ClusterFuzz].
 Fuzzing is a highly effective technique for finding bugs in software.
 ClusterFuzzLite makes your code more secure by fuzzing changes before they even
 enter the codebase.
+
 ClusterFuzzLite supports [GitHub Actions] and [Google Cloud Build], so getting
 the benefits of continuous fuzzing is simple if you can use these CI systems.
 Suppport for more CI systems is in-progress and
-[extending support to other CI systems] is easy.
+[extending support to other CI systems] is easy. ClusterFuzzLite supports programming languages and runtimes
```

Comment:
Change to: "ClusterFuzzLite supports programming languages and runtimes beyond C and C++, including: